api/turing/models/events.go

```diff
@@ -79,3 +79,50 @@ func (e *Event) SetRouter(router *Router) {
 func (e *Event) SetVersion(version uint) {
 	e.Version = version
 }
+
+type EventChannel struct {
```

Comment:
Hmmm.. I think the `models` package has been implicitly reserved for the representation of the DB models. I agree that the event channel related methods were not appropriate for the `utils` package, as they were placed earlier. I wonder if they can be moved into the service layer instead, in `api/turing/service/event_service.go` or even a new package `events` of its own.

Reply:
I moved it out from `utils` package, because (after my changes from this PR) it caused a cyclic import issue. Agree with `event_service.go` as a better place for it, will make a change. Thanks!